_utils/run.sh

```diff
@@ -0,0 +1,2 @@
+python slave_refresh.py
```

Comment:
Lines 1 and 2 need to be swapped.  your want to source the .env before running the refresh job.
Also, remember that the sourced .env stuff won't persist once this file terminates which I guess is OK, if we plan on recreating and resourcing it every time.